### PR TITLE
Add section on filtering

### PR DESCRIPTION
**Pull request by edsko**

### CODE REVIEW

File: wallet-new/docs/spec.tex [hunk 1]
```diff
@@ -25,28 +28,29 @@
 \begin{document}
 
 \title{Cardano wallet specification (DRAFT)}
-\author{Duncan Coutts}
-\date{February 2\textsuperscript{nd} 2018}
+\author{Duncan Coutts \and Edsko de Vries}
```
Comment (dcoutts) [inline wallet-new/docs/spec.tex]: 🎉

File: wallet-new/docs/spec.tex [hunk 2]
```diff
@@ -218,6 +226,36 @@ \subsection{Notation}
...
+
+Both domain restriction and exclusion affect the domain only:
+%
+\begin{align}
+(a \cap \dom u) \restrictdom u & = a \restrictdom u \label{restrictdom_dom} \\
+(a \cap \dom u) \subtractdom u & = a \subtractdom u \label{subtractdom_dom}
+\end{align}
+
+\todo[inline]{Use consistent variable naming.}
+
```
Comment (dcoutts) [inline wallet-new/docs/spec.tex]: These are (improved descriptions of) the lemmas from the other section right? So shouldn't we also delete them from that section and update the label references?

Comment (edsko) [reply wallet-new/docs/spec.tex]: I'm not sure I follow. Are you saying you already stated these lemmas?

Comment (dcoutts) [reply wallet-new/docs/spec.tex]: My mistake. They're different lemmas.

**Pull request merged**